### ISSUE TITLE
Coordinates from example pain analysis do not match source ttl files

### ISSUE THREAD

**Issue by chrisgorgo** [neurostuff/NiMARE#25]
Looking at https://github.com/neurostuff/NiMARE/blob/master/nimare/tests/data/nidm_pain_dset_with_subpeaks.json or https://github.com/neurostuff/NiMARE/blob/master/nimare/tests/data/nidm_pain_dset_with_subpeaks_docker.json or https://github.com/neurostuff/NiMARE/blob/master/nimare/tests/data/nidm_pain_dset.json the coordinates do not match those listed in the source nidm files (looking at the ttl files, for example: https://www.neurovault.org/collections/1425/pain_01.nidm.ttl).

Were the peaks extracted from unthresholded maps instead of nidm files?

**Comment by chrisfilo** [comment]
I'll try to write a function parsing NIDM objects.

Best,
Chris

**Comment by tsalo** [comment]
I'm including the "fixed" dataset jsons in #26, but those can be replaced with ones with peaks extracted from the ttl file once that's ready.

**Comment by chrisfilo** [comment]
What do you mean by "fixed"?

Best,
Chris

**Comment by tsalo** [comment]
I updated the subpeak extraction notebook to use a couple of functions I wrote for `nistats.reporting.get_clusters_table`. The updated version should accurately identify the top 3 sub-peaks for each cluster that are at least 8mm apart from each other and the overall cluster peak. I'm at least more confident with the updated version than I was with the old version of the notebook, which I think I wrote before my PR to update `get_clusters_table` was reviewed, fixed, and merged.

The NIDM ttl file is still the ultimate resource for the coordinates, of course, but the new dataset json could still be useful in the mean time.
